app/services/invoices/view_invoice.service.js

```diff
@@ -6,20 +6,20 @@
 
 const ViewInvoicePresenter = require('../../presenters/view_invoice.presenter.js')
 
-const FetchAndValidateInvoiceService = require('./fetch_and_validate_invoice.service.js')
+const ValidateInvoiceService = require('./validate_invoice.service.js')
 
 class ViewInvoiceService {
   /**
    * Locates and validates an invoice for the specificed bill run and returns the data needed by the View Invoice
```

Comment:
Think we just need to tweak this comment

```suggestion
   * Validates an invoice for the specified bill run and returns the data needed by the View Invoice
```